charts/datadog/ci/security-agent-compliance-values.yaml

```diff
@@ -0,0 +1,12 @@
+datadog:
+  apiKey: "00000000000000000000000000000000"
+  appKey: "0000000000000000000000000000000000000000"
+
+clusterAgent:
+  enabled: true
+
+securityAgent:
```

Comment:
You should add these values to `ci/kubeval.yaml` file (basically this file should have all possible options activated - to validate syntax)